docs/3.0rc/deploy/infrastructure-concepts/deploy-via-python.mdx

```diff
@@ -0,0 +1,385 @@
+        print(f"Hello, {name}!")
+
+    if __name__ == "__main__":
+        my_flow.deploy(
+            name="my-deployment",
+            work_pool_name="my-work-pool",
+            image="my-docker-image:dev",
+            push=False,
+            # Run every weekday at 9am
+            rrule="FREQ=WEEKLY;BYDAY=MO,TU,WE,TH,FR;BYHOUR=9;BYMINUTE=0"
+        )
+    ```
+    </Expandable>
+
+- `schedules`
+
+    Defines multiple schedules for a deployment. This option provides flexibility for:
+        - Setting up various recurring schedules
+        - Implementing complex scheduling logic
+        - Applying timezone offsets to schedules
+
+    <Expandable title="Example">
+    ```python schedules.py
+    from datetime import datetime, timedelta
+    from prefect import flow
+    from prefect.client.schemas.schedules import IntervalSchedule
+
+    @flow(log_prints=True)
+    def my_flow(name: str = "world"):
+        print(f"Hello, {name}!")
+
+    if __name__ == "__main__":
+        my_flow.deploy(
+            name="my-deployment",
+            work_pool_name="my-work-pool",
+            image="my-docker-image:dev",
+            push=False,
+            # Run every 10 minutes starting from January 1, 2023
+            # at 00:00 Central Time
+            schedules=[
+                IntervalSchedule(
+                    interval=timedelta(minutes=10),
+                    anchor_date=datetime(2023, 1, 1, 0, 0),
+                    timezone="America/Chicago"
+                )
+            ]
+        )
+    ```
+    </Expandable>
+
+    Learn more about schedules [here](/3.0rc/automate/add-schedules).
+
+
+## Use remote code storage
+
+In addition to using Docker images, Prefect also supports deploying your code using remote storage. This approach allows you to 
+store your flow code in a remote location, such as a Git repository or cloud storage service.
+
+Using remote storage for your code has several advantages:
+
+1. Faster iterations: You can update your flow code without rebuilding Docker images.
+2. Reduced storage requirements: You don't need to store large Docker images for each code version.
+3. Flexibility: You can use different storage backends based on your needs and infrastructure.
+
+To deploy using remote storage, you'll need to specify where your code is stored by using `flow.from_source` to first load your flow code from a remote location.
+
+Here's an example of loading a flow from a Git repository and deploying it:
+
+```python git-deploy.py
+from prefect import flow
+
+if __name__ == "__main__":
+    flow.from_source(
+        source="https://github.com/username/repository.git",
+        entrypoint="path/to/your/flow.py:your_flow_function"
+    ).deploy(
+        name="my-deployment",
+        work_pool_name="my-work-pool",
+    )
+```
+
+The `source` parameter can be a variety of remote storage options including:
+
+- Git repositories
+- S3 buckets
+- Google Cloud Storage buckets
+- Azure Blob Storage
```

Comment:
could we get an example of what each one of these looks like? (e.g. s3://my-bucket-etc)

Reply:
FWIW, we've got some of this in the flow code storage page, as I recall.